docs/walkthrough/headers.md

```diff
@@ -0,0 +1,43 @@
+# H1 with formatBlock
+
+Let's add a [control button](/concepts/control-button.md) for adding a block-level element around the line containing the current selection:
+```jsx
+import {RichTextContainer, RichTextEditor, useDocumentExecCommand, useDocumentQueryCommandState} from 'bandicoot'
+
+function MyComponent() {
+  return (
+    <RichTextContainer>
+      <H1 value='<h1>'/>
+      <RichTextEditor />
+    </RichTextContainer>
+  )
+}
+
+function H1(props) {
+  const value = {props};
+  const {performCommandWithValue} = useDocumentExecCommand('formatBlock')
```

Comment:
This is out of date now that the `useFormatBlock` hook exists. Additionally, it seems weird to me that this component is called `H1` but you have to pass in a value of `<h1>` to get it to work. Maybe it should be called `Header`?